README.md

```diff
@@ -43,7 +43,7 @@ Add to your app's Build.Gradle:
         mavenCentral()
     }
     dependencies {
-        compile('com.microsoft.identity.client:msal:0.1.1') {
+        compile('com.microsoft.identity.client:msal:0.1.2') {
```

Comment:
@kreedula You forget to change the version number in READ.md

Reply:
Created a PR for change version in readme and its merged to this .